apps/chat/channels.py

```diff
@@ -551,14 +552,19 @@ def _unsupported_message_type_response(self):
         )
 
     def _inform_user_of_error(self):
-        """Simply tells the user that something went wrong to keep them in the loop"""
-        bot_message = self._generate_response_for_user(
-            """
-            Tell the user that something went wrong while processing their message and that they should
-            try again later
-            """
-        )
-        self.send_message_to_user(bot_message)
+        """Simply tells the user that something went wrong to keep them in the loop. This method wil not raise an error
+        if something went wrong during this operation.
+        """
+        try:
+            bot_message = self._generate_response_for_user(
+                """
+                Tell the user that something went wrong while processing their message and that they should
+                try again later
+                """
+            )
+            self.send_message_to_user(bot_message)
+        except Exception as e:
+            logger.exception(f"Something went wrong while trying to inform the user of an error.\{e}")
```

Comment:
In this case should we just send a generic error message to the user. If we move the 'send_message_to_user' call to it's own try catch block then we can handle the errors independently.

Reply:
https://github.com/dimagi/open-chat-studio/pull/1240/commits/23bc1d69f921844432f077cd0f43116d1fe1533e